sql-to-dbsp-compiler/lib/sqllib/src/aggregates.rs

```diff
@@ -3,13 +3,109 @@
 
 use crate::binary::ByteArray;
 use crate::timestamp::*;
+use crate::{FromInteger, ToInteger};
 use core::ops::Add;
-use dbsp::algebra::{F32, F64};
+use dbsp::algebra::{FirstLargeValue, HasOne, HasZero, IsSigned, IsUnsigned, F32, F64};
 use num::PrimInt;
 use rust_decimal::Decimal;
 use std::cmp::Ord;
+use std::fmt::Debug;
 use std::marker::Copy;
 
+/// Holds some methods for wrapping values into unsigned values
+/// This is used by partitioned_rolling_aggregate
```

Comment:
If it's only used by `partitioned_rolling_aggregate`, this code should live near that module. `aggregate.rs` is for regular aggregates only, it's completely separate from the rolling aggregate implementation

Reply:
This is in the sqllib directory